koku/masu/openapi.json

```diff
@@ -812,7 +812,92 @@
         ]
       }
     },
-    "/gcp_invoice_monthly_cost/": {
+    "/db-performance/db-version/": {
+        "get": {
+          "summary": "Returns a table showing the PostgreSQL database software version.",
+          "operationId": "dbPerformanceDbVersion",
+          "description": "Returns a HTML document showing the database version.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing the database version.",
+              "content": "text/html"
+            }
+          },
+          "tags": [
+            "db-performance",
+            "db-version"
+          ]
+        }
+      },
+      "/db-performance/db-settings/": {
+        "get": {
+          "summary": "Returns a table showing the PostgreSQL database software version.",
+          "operationId": "dbPerformanceDbSettings",
+          "description": "Returns a HTML document showing the database software settings.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing the database software settings.",
+              "content": "text/html"
+            }
+          },
+          "tags": [
+            "db-performance",
+            "db-settings"
+          ]
+        }
+      },
+      "/db-performance/stat-statements/": {
+        "get": {
+          "summary": "Returns a table showing query statement statistics if the pg_stat_statements extension has been installed.",
+          "operationId": "dbPerformanceStatStatements",
+          "description": "Returns a HTML document showing query statement statistics if the pg_stat_statements extension has been installed.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing query statement statistics if the pg_stat_statements extension has been installed.",
+              "content": "text/html"
+            }
+          },
+          "tags": [
+            "db-performance",
+            "stat-statements"
+          ]
+        }
+      },
+      "/db-performance/stat-activity/": {
+        "get": {
+          "summary": "Returns a table showing current connection activity.",
+          "operationId": "dbPerformanceStatActivity",
+          "description": "Returns a HTML document showing current connection activity.",
+          "responses": {
+            "200": {
+              "description": "Returns a HTML document showing current connection activity.",
+              "content": "text/html"
```

Comment:
**Issue(blocking):** This does not meet OpenAPI specs: `Structural error at paths./db-performance/stat-activity/.get.responses.200.content should be object`
**Suggestion:** Update the content field to meet OpenAPI specs

Reply:
I added this because people kept asking about that file.
But: If these endpoints are returning HTML instead of JSON, do they really belong in openapi.json?